### PR TITLE
treewide: 21.03 -> 21.05

### PR DESCRIPTION
**Pull request by siraben**
###### Motivation for this change
Per RFC 80

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: nixos/doc/manual/release-notes/rl-2009.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
  <title>Release 20.09 (“Nightingale”, 2020.10/27)</title>
 
   <para>
-   Support is planned until the end of April 2021, handing over to 21.03.
+   Support is planned until the end of April 2021, handing over to 21.05.
```
Comment (dotlambda) [inline nixos/doc/manual/release-notes/rl-2009.xml:9]:
```suggestion
   Support is planned until the end of June 2021, handing over to 21.05.
```

**Comment by SuperSandro2000** [comment]
Why is this rebuilding:

```
aarch64-linux	dotnet-aspnetcore
aarch64-linux	dotnetCorePackages.aspnetcore_2_1
```

?

**Comment by siraben** [comment]
> Why is this rebuilding:
> 
> ```
> aarch64-linux	dotnet-aspnetcore
> aarch64-linux	dotnetCorePackages.aspnetcore_2_1
> ```
> 
> ?

I have no idea. @jonringer could you take a look?

**Comment by zowoq** [comment]
Looks like this is a dupe of #108899?

**Comment by siraben** [comment]
> Looks like this is a dupe of #108899?

Oh it looks like, I wasn't aware of that PR.

**Pull request closed without merging**